CHANGELOG.md

```diff
@@ -1,3 +1,45 @@
+## v0.16.0
```

Comment:
Just flagging to remember v0.17.0

Reply:
The current version is 0.15 so jumping to 0.16 should be sufficient

Reply:
woops, sorry!